components/gitpod-protocol/src/protocol.ts

```diff
@@ -1444,11 +1445,11 @@ export interface OAuth2Config {
 export namespace AuthProviderEntry {
     export type Type = "GitHub" | "GitLab" | string;
     export type Status = "pending" | "verified";
-    export type NewEntry = Pick<AuthProviderEntry, "ownerId" | "host" | "type"> & {
+    export type NewEntry = Pick<AuthProviderEntry, "ownerId" | "host" | "type" | "organizationId"> & {
         clientId?: string;
         clientSecret?: string;
     };
-    export type UpdateEntry = Pick<AuthProviderEntry, "id" | "ownerId"> &
+    export type UpdateEntry = Pick<AuthProviderEntry, "id" | "ownerId" | "organizationId"> &
```

Comment:
nit: actually, I'm no longer sure if `ownerId` is needed here, and same goes for `organizationId`. To update an entry we'd need to identify it, which is done by `id`. That might be a left-over.